Src/Base/AMReX_PlotFileUtil.H

```diff
@@ -2,14 +2,20 @@
 #define AMREX_PLOTFILEUTIL_H_
 #include <AMReX_Config.H>
 
+#include <AMReX_Array.H>
+#include <AMReX_Box.H>
 #include <AMReX_Geometry.H>
+#include <AMReX_IntVect.H>
 #include <AMReX_MultiFab.H>
 #include <AMReX_PlotFileDataImpl.H>
+#include <AMReX_REAL.H>
+#include <AMReX_Vector.H>
```

Comment:
```suggestion
#include <AMReX_Array.H>
#include <AMReX_Geometry.H>
#include <AMReX_MultiFab.H>
#include <AMReX_PlotFileDataImpl.H>
#include <AMReX_Vector.H>
```
It think it's safe to assume they will be "exported" by the MultiFab header.

Reply:
To be more specific, "they" means Box, IntVect, and Real.

Reply:
Ok :+1: